### ISSUE TITLE
[Migration] Read More Link

### ISSUE THREAD

**Issue by wombelix** [getpelican/pelican-plugins#1329]
Hi,

I'm working on the Migration of the [Read More Link](https://github.com/getpelican/pelican-plugins/tree/master/read_more_link) Plugin. 
@justinmayer would you be so kind and create the new Repository in [pelican-plugins](https://github.com/pelican-plugins)?

Thanks in advance :)

Cheers
Dominik

**Comment by justinmayer** [comment]
Hi Dominik! I created the new [Read More](https://github.com/pelican-plugins/read-more) repository and invited you to join it. I must admit I got a bit carried away and pushed the standard set of migration changes to a new `migration` branch, for which I have already created a pull request for your review: https://github.com/pelican-plugins/read-more/pull/1

Looking forward to your comments there. Let's leave this issue open until the migration is marked as completed in the relevant README files in this repository.
